homeassistant/components/whirlpool/sensor.py

```diff
@@ -290,12 +290,15 @@ def update_from_latest_data(self) -> None:
 
         if machine_state is MachineState.RunningMainCycle:
             self._running = True
+
             new_timestamp = now + timedelta(
                 seconds=int(self._wd.get_attribute("Cavity_TimeStatusEstTimeRemaining"))
             )
 
-            if isinstance(self._attr_native_value, datetime) and abs(
-                new_timestamp - self._attr_native_value
-            ) > timedelta(seconds=60):
+            if (
+                isinstance(self._attr_native_value, datetime)
+                and abs(new_timestamp - self._attr_native_value) > timedelta(seconds=60)
+                or self._attr_native_value is None
```

Comment:
It's more efficient to start with the `None` check as that is faster than `isinstance` and Python conditions will short circuit.